lib/tasks/temp.rake

```diff
@@ -456,4 +456,54 @@ namespace :temp do
       Role.where(name: 'notifications_tester').destroy_all
     end
   end
+
+  desc 'Identify broken binary censor rules'
+  task identify_broken_binary_censor_rules: :environment do
+    helper = ApplicationController.helpers
+    helper.class_eval { include Rails.application.routes.url_helpers }
+
+    module ConfigHelper
+      def send_exception_notifications?
+        false
+      end
+    end
+
+    # 0.37.0.0 broken implementation of CensorRule#apply_to_binary
+    # Need to monkeypatch it here to cause the error when searching the affected
+    # attachments.
+    CensorRule.class_eval do
+      def apply_to_binary(binary_to_censor)
+        return nil if binary_to_censor.nil?
+        binary_to_censor.gsub(to_replace('ASCII-8BIT')) do |match|
+          match.gsub(single_char_regexp, 'x')
+        end
+      end
+    end
+
+    ApplicationController.allow_forgery_protection = false
+    app = ActionDispatch::Integration::Session.new(Rails.application)
+
+    CensorRule.find_each do |rule|
+      rule.censorable_requests.find_each do |info_request|
+        next unless info_request.foi_attachments.binary.any?
+
+        info_request.foi_attachments.binary.find_each do |attachment|
+          params =
+            helper.
+            send(:attachment_params, attachment, html: true, only_path: true)
+
+          path = helper.get_attachment_as_html_url(params)
+          protocol = AlaveteliConfiguration.force_ssl ? 'https' : 'http'
+          domain = AlaveteliConfiguration.domain
+          url = "#{ protocol }://#{ domain }#{ path }"
```

Comment:
```suggestion
          url = "#{protocol}://#{domain}#{path}?skip_cache=1"
```

Need to skip the cache otherwise once merged with #5474 some URLs won't be returned if successful response is already cached.

Reply:
Ah good point. Will add a randomised cache key